ChangeLog.d/issue4128.txt

```diff
@@ -0,0 +1,4 @@
+API changes
+   * The option `MBEDTLS_ECP_FIXED_POINT_OPTIM` use pre-computed comb tables
```

Comment:
Minor: the ChangeLog is plain text, so we shouldn't use backticks here. OTOH the migration guide is markdown so backticks are welcome there :)